src/components/features/SignInForm/SignInForm.tsx

```diff
@@ -0,0 +1,31 @@
+"use client";
+import { useState } from "react";
+import { Button } from "../Button/Button";
+import Style from "./SignInForm.module.scss";
+import { InputField } from "../InputField/InputField";
+
+export const SignInForm = () => {
+  const [email, setEmail] = useState("");
+  const [password, setPassword] = useState("");
+  return (
+    <form className={Style.container}>
+      <InputField
+        label="Email"
+        type="email"
+        placeholder="Enter email"
+        id="email"
```

Comment:
If a user type wrong data and submit it, please change inputs statuses and display some info